src/lucky_migrator/create_table_statement.cr

```diff
@@ -118,6 +119,41 @@ class LuckyMigrator::CreateTableStatement
     end
   end
 
+  def default_value(type : String.class, default : String, name)
+    if %w[now() current_time current_timestamp].includes?(default.downcase)
```

Comment:
It seems that a user wouldn't want to set the default time on a string type column, but maybe I just don't know of a use case. Do you have an example of when this might be helpful? Or maybe we can remove this and just use the part on line 126

Reply:
Ah I forgot to remove this when I moved it to line 126